### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kit0-0**
testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+<!-- PROJECT DESCRIPTION -->
+
+<!-- this is my new blog I created with HTML, CSS -->
+
+# Hello Microverse <a name="about-project"></a>
+This project is a Hello Microverse repository.The goal is to master all of the tools and best practices learned in previous steps.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice work you did here in your README file 👏 . However, you're meant to use the [Template](https://github.com/microverseinc/readme-template) provided in the project requirement to customize this section.
- Only the `FAQ` part should be optional
- Every other section should be filled up

## Expected headers:
image

**Review by Shedrack-Sunday — COMMENTED** [review]
Hello @kit0-0      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Nice project setup✔️  
- Linters rightly setup ✔️ 
  
# Required Changes ♻️

- [ ] Firstly, your pull request summary is not descriptive enough. kindly follow [This Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to see a detailed way to do so 👍 

## Your PR view:
image

- [ ] I noticed you merged your PR without approval. going forward, this will lead to invalidation. Always wait for your project to get approved before merging 👍 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (kit0-0) [reply README.md]: @Shedrack-Sunday hi, i updated the description. Do i need to create a new pull request?

Comment (Shedrack-Sunday) [reply README.md]: Hello @kit0-0, please use the same PR while making changes till you get approved

**Pull request merged**